daml/Demo.daml

```diff
@@ -33,6 +33,27 @@ demo = do
 
   issuance providers assets issuer
 
+  let bermudanSwaption floatingRateId fixedRate fixedCcyId principal fixingDates =
```

Comment:
Try putting this in `issuance`, and then change your `daml.yaml` to start that, and watch the script interpreter suffer :laughing: